### DISCUSSION TITLE
Has anyone ever experienced a problem with a clipboard api failing?

### DISCUSSION THREAD

**Discussion by lapse12** [wxt-dev/wxt#893] [Q&A]
I tried the navigator. clipboard.writeText and navigator. clipboard.write api, which work well in the IDE, but I encountered error during testing them
```
TypeError: Cannot read properties of undefined (reading 'writeText')
```

<img width="852" alt="image" src="https://github.com/user-attachments/assets/4319fd89-7f92-418d-8267-64f14857d62f">
<img width="822" alt="image" src="https://github.com/user-attachments/assets/cb09a61c-cc72-4655-afc0-758bd0917ee7">

**Comment by aklinker1** [comment]
For anyone else, follow the github issue for updates: https://github.com/wxt-dev/wxt/issues/894
